app/views/api_configs/_form.haml

```diff
@@ -0,0 +1,11 @@
+.row
+  .col-sm-8.well
+    = f.input :touchpoint_fields, input_html: { value: @config.touchpoint_fields.to_json }
+    = f.input :demographic_fields, input_html: { value: @config.demographic_fields.to_json }
+    = f.input :demographic_fields_with_attributes, input_html: { value: @config.demographic_fields_with_attributes.to_json }
+    = f.input :additional_fields, input_html: { value: @config.additional_fields.to_json }
+    = f.input :identifier
+    = f.input :email
+    = #f.input :password, as: :string
```

Comment:
Is this supposed to be commented out?

Reply:
Nope, it wasn't working yesterday, but it is now.  Thanks.